raiden/tests/utils/events.py

```diff
@@ -127,6 +128,20 @@ def must_have_events(event_list: List[TM], *args) -> bool:
     return True
 
 
+def has_event_of_types(events: List[Event], event_types: List[Type]) -> bool:
```

Comment:
Could you also make the other utilities work on tuples? (Perhaps that would be better done on a follow up)

Reply:
I'll add it to the pr for #4524 then, so this one doesn't take forever.